src/frontend/shortlist/src/components/school/SchoolCard.vue

```diff
@@ -26,40 +27,44 @@ export default {
     <div class="school-simple-container" @click="swapDetail">
       <template v-if="!isDetail">
         <div class="school-simple-img-container">
-          <img
-            class="school-simple-img"
-            :src="schoolData.img"
-            draggable="false"
-          />
+          <img class="school-simple-img" :src="defaultImg" draggable="false" />
+          <!-- :src="schoolData.img" -->
         </div>
       </template>
 
       <div class="school-simple-name-row">
         <div class="school-simple-name-name">
           <template v-if="isDetail">
-            <a :href="schoolData.link">{{ schoolData.name }}</a>
+            <a :href="schoolData.school.schoolMetadata.link">{{
+              schoolData.school.schoolMetadata.name
+            }}</a>
           </template>
-          <template v-if="!isDetail">
-            {{ schoolData.name }}
+          <template v-else>
+            {{ schoolData.school.schoolMetadata.name }}
```

Comment:
Is moving things from schoolData to Metadata also part of the integration or moving from one database to another?